CHANGELOG.md

```diff
@@ -75,10 +83,6 @@ All notable changes to this project will be documented in this file. The format
 - \(FM-8194\) Convert tests to litmus [\#156](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/156) ([eimlav](https://github.com/eimlav))
 - \(MODULES-9224\) Add no progress flag [\#154](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/154) ([michaeltlombardi](https://github.com/michaeltlombardi))
 
-### Fixed
```

Comment:
Although [this PR](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/147) was tagged with the `bugfix` label, it seems to be under the **Added** section [above](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/238/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL75) too.